### PR TITLE
Fix monitoring test

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Fix monitoring test

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/10801

### CODE REVIEW

**Review by themue — DISMISSED** [review]

File: resources/eventing/charts/publisher-proxy/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -16,5 +14,4 @@ spec:
       - {{ .Release.Namespace }}
   endpoints:
     - port: {{ .Values.metrics.config.portName }}
-      interval: {{ .Values.metrics.config.interval }}
-{{- end }}
\ No newline at end of file
+      interval: {{ .Values.metrics.config.interval }}
```
Comment (radufa) [inline resources/eventing/charts/publisher-proxy/templates/servicemonitor.yaml:17]: nit: add new line

**Review by radufa — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Pull request merged**